docs/server-core/python-core.mdx

```diff
@@ -59,6 +60,7 @@ export const Builder = SDKDocsBuilder({
         install: <Install />,
         preamble: <Preamble />,
         initialize: <Initialize />,
+        sharedInstance: <SharedInstance />,
```

Comment:
Move all the shared instance documentation to the "WorkingWith" section. You will also need to update the "WorkingWIth" template to actually render the sharedInstance snippet